content/guides/compute.md

```diff
@@ -4,3 +4,331 @@ title: Compute Engine documentation | ARGO
+        "service_type_A":"OR",
+        "service_type_B":"OR"
+        },
+       "operation":"OR"
+    },
+    "my_group_of_services_2": {
+      "services":{
+        "service_type_C":"OR",
+        "service_type_D":"OR"
+        },
+       "operation":"OR"
+    },
+"operation":"AND"
+```
+
+In the above example the service types are grouped in two groups: ***my_group_of_services_1*** and ***my_group_of_services_2***. Each group contains a "service" list containing service types included in the group as fields and the operation values in orded to choose who to aggregate the various instances of a specific service. For example if for ***"service_type_A"*** are 3 service endpoints available they are goint to be aggregated using the OR operation. The ***"operation"*** field under each group of services is used to declare the operation that will be used to aggregate the service types under that group.   
+The outer ***"operation"*** field in the root of the json document is used to declare the operation used to aggregate the various groups in order to produce the final endpoint aggregation result.
+
+
+
+
+
+
+
+### Executable Scripts of the standalone edition
+In the folder `/usr/libexec/ar-compute/standalone/` reside executable scripts that can be used for uploading metric data and sync data to the hadoop cluster (HDFS Filesystem). 
+##### upload_metric.py 
+The specific script is used in order to upload daily metric data (relative to a tenant) to HDFS.   
+full path: `/usr/libexec/ar-compute/standalone/upload_metric.py`
+
+parameters:
+- `-d --date {YYYY-MM-DD}`  
+specifies the date of the metric data we want to upload
+- `-t --tenant {STRING}`  
+a case-sensitive string specifing the name of the tenant
+
+
+##### upload_sync.py
+The specific script is used in order to upload daily sync data (relative to a tenant and a job) to HDFS.
+full path: `/usr/libexec/ar-compute/standalone/upload_sync.py` 
+
+parameters:
+- `d --date {YYYY-MM-DD}`  
+specifies the date of the sync data we want to upload  
+- `t --tenant {STRING}`  
+a case-sensitive string specifing the name of the tenant  
+- `j --job {STRING}`  
+a case-sensitive string specifing the name of the job
+
+##### mongo_clean_ar.py
+The specific script is used if necessary to clean a/r data from the datastore regarding a specific day
+full path: `/usr/libexec/ar-compute/standalone/mongo_clean_ar.py` 
+
+parameters:  
+- `d --date {YYYY-MM-DD}`  
+specifies the date (day) to clear the data
+
+optional:
+- `p --profile {STRING}`  
+specificy the name of an availability profile. If specified only data a/r data regarding the specified profile will be cleared
+
+##### mongo_clean_status.py
+The specific script is used if necessary to clean status detail data from the datastore regarding a specific day
+full path: `/usr/libexec/ar-compute/standalone/mongo_clean_status.py` 
+
+parameters:  
+- `d --date {YYYY-MM-DD}`  
+specifies the date (day) to clear the data
+
+
+##### job_ar.py
```

Comment:
It is not clear what this script does and how it is different from the other job scripts